src/views/workflow-queries/workflow-queries-result/workflow-queries-result.types.ts

```diff
@@ -8,7 +8,12 @@ export type Props = {
   loading?: boolean;
 };
 
+export type Markdown = {
```

Comment:
Instead of adding new types of content on a case-by-case basis, we can have a generic "custom content" type and add a configuration that parses it based on what the "type" field is.